src/app/components/article/ArticlesSidebar.js

```diff
@@ -0,0 +1,225 @@
+    createExplainerItem(input: $input) {
+      project_media {
+        id
+        fact_check {
+          id
+        }
+        explainers(first: 100) {
+          edges {
+            node {
+              id
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const addFactCheckMutation = graphql`
+  mutation ArticlesSidebarUpdateClaimDescriptionMutation($input: UpdateClaimDescriptionInput!) {
+    updateClaimDescription(input: $input) {
+      project_media {
+        id
+        fact_check {
+          id
+        }
+        explainers(first: 100) {
+          edges {
+            node {
+              id
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const ArticlesSidebarComponent = ({
+  team,
+  projectMedia,
+  onUpdate,
+}) => {
+  const [adding, setAdding] = React.useState(false);
+  const setFlashMessage = React.useContext(FlashMessageSetterContext);
+  const hasArticle = (projectMedia.fact_check?.id || projectMedia.explainers.edges[0]?.node?.id);
+
+  const onCompleted = () => {
+    setFlashMessage(
+      <FormattedMessage
+        id="articlesSidebar.success"
+        defaultMessage="Article added successfully!"
+        description="Banner displayed after an article is successfully added to an item."
+      />,
+      'success');
+    setAdding(false);
+    onUpdate();
+  };
+
+  const onError = (error) => {
+    const errorMessage = getErrorMessage(error);
+    const errorComponent = errorMessage || <GenericUnknownErrorMessage />;
+    setFlashMessage(errorComponent);
+    setAdding(false);
+  };
+
+  const handleAdd = (nodeType, id) => { // "id" should be an explainer database ID or a fact-check claim description GraphQL ID
+    if (!adding) {
+      setAdding(true);
+      let input = null;
+      let mutation = null;
+      if (nodeType === 'Explainer') {
+        mutation = addExplainerMutation;
+        input = {
+          explainerId: id,
+          projectMediaId: projectMedia.dbid,
+        };
+      } else if (nodeType === 'FactCheck') {
+        mutation = addFactCheckMutation;
+        input = {
+          id,
+          project_media_id: projectMedia.dbid,
+        };
+      }
+      commitMutation(Relay.Store, {
+        mutation,
+        variables: {
+          input,
+        },
+        onCompleted,
+        onError,
+      });
+    }
+  };
+
+  return (
+    <div id="articles-sidebar" className={styles.articlesSidebar}>
+      <div className={styles.articlesSidebarTopBar}>
+        {/* FIXME: Make sure the form can receive the right reference for the current item */}
+        <NewArticleButton team={team} buttonMainProps={{ size: 'small', theme: 'text' }} disabled={projectMedia.type === 'Blank'} />
+      </div>
+      { !hasArticle && (
+        <>
+          <div className={cx('typography-body1', styles.articlesSidebarNoArticle)}>
+            <DescriptionIcon style={{ fontSize: '32px' }} />
```

Comment:
Maybe we're missing a matching `iconSize...` css variable @brianfleming

Reply:
Good catch @amoedoamorim 
@caiosba you can change this and put it in the css file and use the var:

`font-size: var(--font-size-h4);`

Reply:
@brianfleming `--font-size-h4` is 34px while the icon on the design was 32px - is it OK to keep it 34px?

Reply:
yes! I checked it in the Figma design file and it works fine

Reply:
Thanks, done!